### PR TITLE
Wrap math content in `<script type="math/tex"></script>` tags for MathJax

### PR DESCRIPTION
**Pull request by drgrice1**
This is a replacement for @Alex-Jordan's previous attempt #563.

This is paired with openwebwork/webwork2#1322 and won't work without it.

### CODE REVIEW

**Review by Alex-Jordan — APPROVED** [review]
These work for me.

There is a bonus improvement. Before when the page first loads, you see the unprocessed math content for a second before MathJax processes. With this, the script tag content is not visible until the math is processed. I think that is an improvement, although it's worth noting that the rest of the text will tend to move more now that there is not unprocessed math serving as a placeholder.

**Review by pstaabp — APPROVED** [review]

File: macros/PGbasicmacros.pl [hunk 1]
```diff
@@ -2112,7 +2112,7 @@ sub general_math_ev3 {
 
 	my $out;
 	if($displayMode eq "HTML_MathJax") {
-		$out = '<span>'.$in_delim.'</span>';
+		$out = '<span><script type="math/tex' . ($mode eq 'display' ? '; mode=display' : '') . '">' . $in . '</script></span>';
```
Comment (dpvc) [inline macros/PGbasicmacros.pl:2115]: Is there a need for the surrounding `<span>`?  I don't see one myself.

**Review by dpvc — APPROVED** [review]
One unnecessary `<span>`, but other than that, OK.

File: macros/PGbasicmacros.pl (hunk 1 above)
Comment (drgrice1) [reply macros/PGbasicmacros.pl:2115]: It was there before, and I just left it.   It isn't needed as far as I can tell.  I removed it.

**Pull request merged**